working-groups/meta/NOTES.md

```diff
@@ -1,6 +1,42 @@
 # Meta Meeting Notes
 This document contains meeting notes from the Meta working group.
 
+## 2019-02-28: [Meeting][meeting20190228]
+**Written by:** [@spastorino][spastorino]
+[spastorino]: https://github.com/spastorino
+### Roles of journeypeople and team members (questions raised)
+
+- What problems do we want to solve from a compiler team member perspective and from a contributor perspective?
+  - Help to grow the team by ensuring that we engage people who've started to get active in rustc?
+  - Provide a source of people who can help implement things without needing as much mentoring?
+
+- What does it mean to be a journeyperson?
+  - You know something about the compiler?
+  - People willing to commit some time to help run things
+  - What is the role of a journeyperson in onboarding new compiler contributors?
+  - r+ rights?
+  - Start to do reviews?
+  - What responsibilities should a journeyperson have?
+  - Are journeypeople members of the team or is a journeyperson role considered a stepping stone toward being a member of the team?
+
+
+- What are the qualifications and expectations for a member of the compiler team?
+
+- What's the difference between journeypeople and team members?
```

Comment:
rfcbot powers seems like an important distinction? Notably, if you have too many team members, rfcbot will likely get stuck in limbo on all sorts of issues and decisions that need to be made.

Reply:
(while potentially true, this is meant to be minutes from the meeting that occurred)

Reply:
Oh; right... forgot ^^